corgi/collectors/errata_tool.py

```diff
@@ -209,7 +212,7 @@ def get_variant_cdn_repo_mapping() -> dict[str, list[str]]:
         }
         return variants_with_repos
 
-    def get_errata_key_details(self, name: str) -> tuple[int, bool]:
+    def get_errata_key_details(self, name: str) -> tuple[int, bool, bool]:
```

Comment:
observation - always difficult to know if a function returning tuples is better then just atomic getters ... both approaches can complexify other code ... this is fine, just opining.